EIPS/eip-1959.md

```diff
@@ -0,0 +1,70 @@
+---
+eip: 1959
+title: Valid ChainID opcode
```

Comment:
I think this could be improved to describe the proposal better. Only reading the title I've immediatly thought it is the same as 1344.